docs/pages/web/overview.js

```diff
@@ -113,15 +122,17 @@ function List({ array, title = '' }: {| array: ListItemType, title?: string |}):
 }
 
 export default function ComponentOverview(): Node {
-  const [order, setOrder] = useState('category');
+  const [order, setOrder] = useState('alphabetical');
```

Comment:
We don't want to start with categorical organization here? We already default to alphabetical organization in the sidenav (well, it's the only option we offer 😅) — would it not make sense to keep the default here by category? I think that also makes the page more browsable, which is the goal of the page.

Reply:
@hectoid  Thoughts?